libs/experimental/langchain_experimental/graph_transformers/llm.py

```diff
@@ -576,13 +576,17 @@ def _convert_to_graph_document(
     else:  # If there are no validation errors use parsed pydantic object
         parsed_schema: _Graph = raw_schema["parsed"]
         nodes = (
-            [map_to_base_node(node) for node in parsed_schema.nodes]
+            [map_to_base_node(node) for node in parsed_schema.nodes if node.id]
```

Comment:
Should we also test for empty label?

Reply:
If empty label is passed it uses default "Node" label

Reply:
So I wouldn't skip records if node labels are missing, as we just use default node label in that case